CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Dropped support for Helm V2 and converted to Helm V3.
   [#60](https://github.com/cyberark/sidecar-injector/pull/60)
 
+### Security
+- Updated golang.org/x/net to v0.0.0-20211209124913-491a49abca63
+  [cyberark/sidecar-injector#63](https://github.com/cyberark/sidecar-injector/pull/63)
```

Comment:
Looks like the changelog parser is failing on this line. I think there's not supposed be a newline before the hyperlink. Also looks like there are some tests failing which I think has to do with Golang versions. I'm working on a PR to update this repo to Golang 1.17 which should fix this issue in the process.

Reply:
Here it is: #65